packages/riverpod_generator/lib/src/models.dart

```diff
@@ -16,14 +16,16 @@ enum ProviderType {
   asyncNotifier,
 }
 
+enum AsyncType { future, futureOr, consecutive }
```

Comment:
What is "consecutive" supposed to mean in this context?

Reply:
What about having a "ValueType" provider with "future, futureOr, value"

And in the future new values such as "stream" could be added

Reply:
I'll admit I'm consfused by your questions. I'm not sure how they related to the discussion at hand.

I only talked about renaming the enum class and the associated variable; as I believe "AsyncType" is not an appropriate name